kv/db.go

```diff
@@ -123,7 +111,8 @@ func (s *DBServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	method = strings.TrimPrefix(method, DBPrefix)
-	if !proto.IsPublic(method) {
```

Comment:
D'oh, ignore that last sentence. I saw the type name DBServer and thought this was in the server package.